<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RLCOM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ICON/icon.qrc">
    <normaloff>:/new/prefix1/logo.png</normaloff>:/new/prefix1/logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="LogoLab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>200</horstretch>
        <verstretch>200</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ICON/icon.qrc">:/new/prefix1/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="ProtConfigure">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>串口配置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="3" column="1">
        <widget class="QComboBox" name="CheckBitCbox">
         <item>
          <property name="text">
           <string>无校验</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>偶校验</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>奇校验</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="BaudLab">
         <property name="text">
          <string>波特率：</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="SwitchPortPbtn">
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="StopBitCbox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="BaudCbox">
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>460800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>576000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>921600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1152000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1500000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2000000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2500000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3000000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3500000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4000000</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="CheckBitLab">
         <property name="text">
          <string>校验位：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="DataBitCbox">
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="PortCbox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="StopBitLab">
         <property name="text">
          <string>停止位：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="DataBitLab">
         <property name="text">
          <string>数据位：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="PortLab">
         <property name="text">
          <string>端  口：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="CleanStatsPbtn">
         <property name="text">
          <string>清空统计</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QTabWidget" name="TabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Tab1">
       <attribute name="title">
        <string>串口收发</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="ReceiveGbox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>7</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>接收区</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QCheckBox" name="HexShowCbox">
             <property name="text">
              <string>十六进制显示</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="AutoFollowCbox">
             <property name="text">
              <string>自动跟踪</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QCheckBox" name="TalkWindowCbox">
             <property name="text">
              <string>对话式窗口</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="CleanReceivePbtn">
             <property name="text">
              <string>清空接收区</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="4">
            <widget class="QTextEdit" name="ReceiveTextEdit">
             <property name="lineWrapMode">
              <enum>QTextEdit::WidgetWidth</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::NoTextInteraction</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="SendGbox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>3</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>发送区</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="7">
            <widget class="QPushButton" name="SendPbtn">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="EnterSendCBox">
             <property name="text">
              <string>回车键发送</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QSpinBox" name="TimeSBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
             <property name="value">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QPushButton" name="CleanSendPbtn">
             <property name="text">
              <string>清空发送区</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QCheckBox" name="AutoEnterCBox">
             <property name="text">
              <string>自动换行</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QCheckBox" name="TimeSendCbox">
             <property name="text">
              <string>定时发送（单位ms）</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="HexSendCbox">
             <property name="text">
              <string>十六进制发送</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QCheckBox" name="SendCleanCbox">
             <property name="text">
              <string>发送后清除</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="8">
            <widget class="QTextEdit" name="SendTextEdit">
             <property name="acceptRichText">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Tab2">
       <attribute name="title">
        <string>图像传输</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>图像配置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="3" column="0" colspan="2">
            <widget class="QCheckBox" name="GridCbox">
             <property name="text">
              <string>图像栅格</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="WidthLab">
             <property name="text">
              <string>Width：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="WidthLedit">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="SwitchImagePbtn">
             <property name="text">
              <string>开始接收</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QComboBox" name="ImageModeCBox">
             <item>
              <property name="text">
               <string>灰度图</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>二值化图</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="HeightLedit">
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="HeightLab">
             <property name="text">
              <string>Height：</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ImageGbox">
          <property name="title">
           <string>Image</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="ImageLab">
             <property name="text">
              <string>Waiting......</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Tab3">
       <attribute name="title">
        <string>虚拟示波</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="1" column="1">
         <widget class="QCustomPlot" name="ChartWidget" native="true"/>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QGroupBox" name="WaveformConfigure">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>波形配置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="1" column="1">
            <widget class="QComboBox" name="DataTypeCbox">
             <item>
              <property name="text">
               <string>uint8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>uint16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>uint32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>int8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>int16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>int32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>float</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>double</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="ChannleSbox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="SwitchWaveformPbtn">
             <property name="text">
              <string>开始接收</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="DataTypeLab">
             <property name="text">
              <string>数据类型：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="ChannleLab">
             <property name="text">
              <string>通道数：</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="ChartConfigure">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>140</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>140</height>
           </size>
          </property>
          <property name="title">
           <string>图表配置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QCheckBox" name="AdaptiveCbox">
             <property name="text">
              <string>自适应窗口</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="ChartAutoFollowCbox">
             <property name="text">
              <string>自动跟踪</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="XaxisZoomPbtn">
             <property name="text">
              <string>X轴放大</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="XaxisShrinkPbtn">
             <property name="text">
              <string>X轴缩小</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ICON/icon.qrc"/>
 </resources>
 <connections/>
</ui>
